packages/cli/lang/en.lyaml

```diff
@@ -993,6 +997,13 @@ en:
         logFeedbackMessage:
           feedbackHeader: "We'd love to hear your feedback!"
           feedbackMessage: "How are you liking the new projects and developer tools? \n > Run `{{#yellow}}hs feedback{{/yellow}}` to let us know what you think!\n"
+      publicApps:
```

Comment:
@markhazlewood, if I get could your feedback on these error messages, I'd be very appreciative. Connected [Loom](https://www.loom.com/share/1476c63cc3814ce5bfe8330dda699e0d) for your convenience.

Reply:
Canceling this request for comment. Going to rework the commands after the Projects UX sync today.